packages/plugin-ext/src/hosted/node/hosted-plugin-process.ts

```diff
@@ -184,7 +189,7 @@ export class HostedPluginProcess implements ServerPluginRunner {
             silent: true,
             env: env,
             execArgv: [],
-            stdio: ['pipe', 'pipe', 'pipe', 'ipc']
+            stdio: ['pipe', 'pipe', 'pipe', 'ipc', 'pipe']
```

Comment:
For this change to have a chance to work on Windows:

```suggestion
            stdio: ['pipe', 'pipe', 'pipe', 'ipc', 'overlapped']
```